apps/sms/js/thread_list_ui.js

```diff
@@ -392,7 +394,7 @@ var ThreadListUI = {
   markReadUnread: function thlui_markReadUnread(selected, isRead) {
     selected.forEach((id) => {
       var thread  = Threads.get(id);
-      var markable = thread && (!thread.hasDrafts || isRead);
+      var markable = thread && (isRead || !thread.hasDrafts);
```

Comment:
I'm not sure how it's done in JS, but usually it's faster when in such conditions "the fastest" property is placed first because the rest of args is ignored. Do you know if it's true for JS